source/common/tracing/http_tracer_impl.h

```diff
@@ -27,6 +27,54 @@ struct Decision {
   bool is_tracing;
 };
 
+/**
+ * Tracing tag names.
+ */
+class TracingTagValues {
+public:
+  // OpenTracing standard tag names.
+  const std::string COMPONENT = "component";
+  const std::string DB_INSTANCE = "db.instance";
+  const std::string DB_STATEMENT = "db.statement";
+  const std::string DB_USER = "db.user";
+  const std::string DB_TYPE = "db.type";
+  const std::string ERROR = "error";
+  const std::string HTTP_METHOD = "http.method";
+  const std::string HTTP_STATUS_CODE = "http.status_code";
+  const std::string HTTP_URL = "http.url";
+  const std::string MESSAGE_BUS_DESTINATION = "message_bus.destination";
+  const std::string PEER_ADDRESS = "peer.address";
+  const std::string PEER_HOSTNAME = "peer.hostname";
+  const std::string PEER_IPV4 = "peer.ipv4";
+  const std::string PEER_IPV6 = "peer.ipv6";
+  const std::string PEER_PORT = "peer.port";
+  const std::string PEER_SERVICE = "peer.service";
+  const std::string SPAN_KIND = "span.kind";
+
+  // Non-standard tag names.
+  const std::string DOWNSTREAM_CLUSTER = "downstream_cluster";
+  const std::string GRPC_STATUS_CODE = "grpc.status_code";
+  const std::string GUID_X_CLIENT_TRACE_ID = "guid:x-client-trace-id";
+  const std::string GUID_X_REQUEST_ID = "guid:x-request-id";
+  const std::string HTTP_PROTOCOL = "http.protocol";
+  const std::string NODE_ID = "node_id";
+  const std::string REQUEST_SIZE = "request_size";
+  const std::string RESPONSE_FLAGS = "response_flags";
+  const std::string RESPONSE_SIZE = "response_size";
+  const std::string STATUS = "status";
+  const std::string UPSTREAM_CLUSTER = "upstream_cluster";
+  const std::string UPSTREAM_CLUSTER_NAME = "upstream_cluster_name";
```

Comment:
@mattklein123 Think this one should be removed in favour of the `UPSTREAM_CLUSTER` - I'll create in separate PR in case any users have objections?

Reply:
SGTM